arrow/src/main/java/org/apache/iceberg/arrow/vectorized/NullabilityHolder.java

```diff
@@ -52,10 +52,12 @@ public void setNotNull(int index) {
   }
 
   public void setNulls(int startIndex, int num) {
+    // TODO: do we have to modify numNulls?
     System.arraycopy(nulls, 0, isNull, startIndex, num);
   }
 
   public void setNotNulls(int startIndex, int num) {
+    // TODO: do we have to modify numNulls?
```

Comment:
Unless we were unsetting a value that was previously set as null I don't think we would change things here. This api basically requires you to only call "set" on an index a single time.

Reply:
Yeah, agree.

Reply:
Not applies anymore.